packages/pools/src/constants/pools/1.ts

```diff
@@ -4,6 +4,14 @@ import { getAddress } from 'viem'
 import { PoolCategory, SerializedPool } from '../../types'
 
 export const livePools: SerializedPool[] = [
+  {
+    sousId: 10,
+    stakingToken: ethereumTokens.cake,
+    earningToken: ethereumTokens.roci,
+    contractAddress: '0x50e0D135154712c0514446b25B5773BEe32709Fa',
+    poolCategory: PoolCategory.CORE,
+    tokenPerSecond: '0.2353',
+  },
```

Comment:
let's remove it. Also remove the one in farm config

Reply:
got it